.github/workflows/ci.yml

```
@@ -241,7 +245,8 @@ jobs:
           universal: "false"
           dist-folder: dist
           output-folder: dist-artifacts
-          access-policy-token: ${{ env.SIGN_PLUGIN_ACCESS_POLICY_TOKEN }}
+          access-policy-token: ${{ env.SIGN_PLUGIN_ACCESS_POLICY_TOKEN || '' }}
+          allow-unsigned: ${{ env.IS_FORK }}
```

Maybe we should be super clear here
```suggestion
          allow-unsigned: ${{ env.IS_FORK == 'true' }}
```